docs/theory/dft/functional_derivatives.md

```diff
@@ -0,0 +1,39 @@
+## Functional derivatives
+
+In the last section the functional derivative
+
+$$\hat F_{\rho_\alpha}^\mathrm{res}(r)=\left(\frac{\delta F^\mathrm{res}}{\delta\rho_\alpha(r)}\right)_{T,\rho_{\alpha'\neq\alpha}}$$
```

Comment:
Missing `\hat` in derivative?

`$$\hat F_{\rho_\alpha}^\mathrm{res}(r)=\left(\frac{\delta \hat F^\mathrm{res}}{\delta\rho_\alpha(r)}\right)_{T,\rho_{\alpha'\neq\alpha}}$$`